generator/src/main/java/com/scottlogic/deg/generator/config/detail/VisualiserLevel.java

```diff
@@ -0,0 +1,26 @@
+package com.scottlogic.deg.generator.config.detail;
+
+public enum VisualiserLevel  {
```

Comment:
this never really seems to be used, is it removable?

Reply:
this class is used.  if user does not specify --visualiser-level it defaults to OFF, otherwise if they specify standard it will log out the initial, pruned, optimised and partitioned trees.  if specify DETAILED it will also print out trees throughout each prune done during the walking.  This PR does not do all of this yet.  This will be in follow up PRs.  This PR will include hwta done so far, the fixes you suggested and updates to the DOCs (which not done yet)